src/lib/ng-v5/entry-point/resources/stylesheet.transform.ts

```diff
@@ -57,13 +57,25 @@ async function renderPostCss(filePath: string, cssStyles: string, cssUrl: CssUrl
   log.debug(`determine browserslist for ${filePath}`);
   const browsers = browserslist(undefined, { filePath });
 
-  const postCssPlugins = [autoprefixer({ browsers }), postcssComments({ removeAll: true })];
+  const postCssPlugins = [];
 
   if (cssUrl !== CssUrl.none) {
     log.debug(`postcssUrl: ${cssUrl}`);
     postCssPlugins.push(postcssUrl({ url: cssUrl }));
   }
 
+  // this is important to be executed post running `postcssUrl`
+  postCssPlugins.push(
+    autoprefixer({ browsers }),
+    postCssClean({
+      level: {
+        2: {
+          specialComments: false
```

Comment:
See list of optoimization at css-clean docs: https://github.com/jakubpawlowicz/clean-css#level-2-optimizations

Reply:
Yeah those are on by default, just wanted to sdd this additional option which at the moment is done via another plugin.